documentstore_migracao/utils/convert_html_body.py

```diff
@@ -1073,6 +1094,7 @@ def __init__(self):
             self.TableinBody(),
             self.TableinP(),
             self.AddPinFN(),
+            self.WrapNodeIDefItem(),
         )
```

Comment:
O WrapNodeIDefItem não deveria estar dentro do tratamento "dd" ? 
IPipe, se não me engano é itálico. E acho que não há relação com dd. Se tem foi por acaso.

Reply:
@robertatakenaka , Escrevi a assinatura errada da classe, eu queria disser `WrapNodeInDefItem`, ja que a ideia desse pipe line e envelopar as tags filhas do `def-item`, criando tags `def` ou tag `term` caso no `def-item` haja `.text` ou `.tail`